src/antic/nf_elem.jl

```diff
@@ -396,11 +396,11 @@ function -(a::QQFieldElem, b::AbsSimpleNumFieldElem)
   return r
 end
 
-+(a::AbsSimpleNumFieldElem, b::Integer) = a + ZZRingElem(b)
++(a::AbsSimpleNumFieldElem, b::Integer) = a + flintify(b)
```

Comment:
test coverage for adhoc operations is really bad. Would be good to have more of those in the interface tests (possibly optionally, as I am not sure we mandate this in the ring interface?)

Reply:
the ring interface states that it is optional to add explicit ad-hoc functions, but AA should take care of the fallback using promotion (see https://nemocas.github.io/AbstractAlgebra.jl/dev/ring_interface/#Optional-binary-ad-hoc-operators)